core/structs.go

```diff
@@ -4,9 +4,10 @@ import "C"
 
 type Module struct {
 	Name    string `json:"name"`
+	Workdir string
 	Type    string `json:"type"`
 	Modules []map[string]interface{}
-	Content []byte // The entire module unparsed as a []byte, used by plugins
```

Comment:
Did you mean to change this line?

Reply:
Hello Tau. Thank you for pointing this out. Seems like the comment was automatically removed by my autoformatting config. The issue seems to be with the `goimports-reviser` so I have a switched to a different formatter for now. I will revert the change in an upcoming commit.